### PR TITLE
Gardening: Migrate test suite to GH issues p. 16

### PR DESCRIPTION
**Pull request by AnthonyLatsis**

### CODE REVIEW

File: test/IRGen/unmanaged_objc_throw_func.swift [hunk 1]
```diff
@@ -4,12 +4,14 @@
...
+
+@objc protocol P {
   func returnUnmanagedCFArray() throws -> Unmanaged<CFArray>
 }
 
-// CHECK-LABEL: define hidden swiftcc %TSo10CFArrayRefa* @"$s25unmanaged_objc_throw_func9SR_9035_CC22returnUnmanagedCFArrays0G0VySo0H3RefaGyKF"
-@objc class SR_9035_C: NSObject, SR_9035_P {
+// CHECK-LABEL: define hidden swiftcc %TSo10CFArrayRefa* @"$s25unmanaged_objc_throw_func1CC22returnUnmanagedCFArrays0F0VySo0G3RefaGyKF"
+@objc class C: NSObject, P {
   func returnUnmanagedCFArray() throws -> Unmanaged<CFArray> {
```
Comment (AnthonyLatsis) [inline test/IRGen/unmanaged_objc_throw_func.swift:15]: The slight divergence in the `Unmanaged` and `CFArray` result portions of the mangled signature, `s0G0V → s0F0V` and `So0H3Refa → So0G3Refa` respectively, which exercise [word substitutions](https://github.com/apple/swift/blob/main/docs/ABI/Mangling.rst#identifiers), are expected and due to `C` having one less word—none, in fact—than the old `SR_9035_C`.

File: test/IRGen/big_types_corner_cases.swift [hunk 2]
```diff
@@ -261,76 +261,77 @@ class ClassWithLoadableStructWithBiggerString {
...
+    func forceHandle_1<Q: Query>(query: Q) -> Void {
         guard let body = handle_1 as? (Q) -> Void else {
             fatalError("handler \(self) is expected to handle query \(query)")
         }
         body(query)
     }
 
-// CHECK-LABEL: define{{( dllexport)?}}{{( protected)?}} swiftcc void @"$s22big_types_corner_cases19sr8076_QueryHandlerPAAE13forceHandle_25query8ReturnedQyd___AA9BigStructVSgtqd___tAA0e1_F0Rd__lF"(%T22big_types_corner_cases9BigStructVSg* noalias nocapture sret({{.*}}) %0, %swift.opaque* noalias nocapture %1, %swift.opaque* noalias nocapture %2, %swift.type*{{.*}}, %swift.type*{{.*}}, i8** {{.*}}.sr8076_QueryHandler, i8** {{.*}}.sr8076_Query, %swift.opaque* noalias nocapture swiftself %3)
+// CHECK-LABEL: define{{( dllexport)?}}{{( protected)?}} swiftcc void @"$s22big_types_corner_cases12QueryHandlerPAAE13forceHandle_25query8ReturnedQyd___AA9BigStructVSgtqd___tAA0E0Rd__lF"(%T22big_types_corner_cases9BigStructVSg* noalias nocapture sret({{.*}}) %0, %swift.opaque* noalias nocapture %1, %swift.opaque* noalias nocapture %2, %swift.type*{{.*}}, %swift.type*{{.*}}, i8** {{.*}}.QueryHandler, i8** {{.*}}.Query, %swift.opaque* noalias nocapture swiftself %3)
```
Comment (AnthonyLatsis) [inline test/IRGen/big_types_corner_cases.swift:301]: Ditto these `CHECK-LABEL` directives.

**Comment by xedin** [comment]
Sorry I am behind with this, I will try to review at least 3 PRs tomorrow.

**Comment by AnthonyLatsis** [comment]
Thanks, Pavel! Please don't feel obliged to keep up with the pace at which I'm spawning these PRs though.

**Review by xedin — APPROVED** [review]

**Comment by AnthonyLatsis** [comment]
@swift-ci please test

**Pull request merged**